solutions_builder/template_root/terraform/stages/2-foundation/main.tf

```diff
@@ -62,6 +62,10 @@ locals {
 # Used to retrieve project_number later
 data "google_project" "project" {}
 
+resource "google_identity_platform_config" "default" {
+  project = var.project_id
```

Comment:
why is this added in this PR?

Reply:
This was from the previous PR to enable identity platform. I will remove it